internal/parse/parse.go

```diff
@@ -39,6 +39,11 @@ func MustParseModule(policy string) *ast.Module {
 	return ast.MustParseModuleWithOpts(policy, ParserOptions())
 }
 
+// ParseModule works like ast.ParseModule but with the Regal parser options applied.
+func ParseModule(policy string) (*ast.Module, error) {
```

Comment:
This is not public, should it be? It's hard to manually build data for with NewInput without these, but InputFromText and InputFromPaths might make exposing it surplus.

Reply:
Let’s hide it for now then.

Reply:
👍